### PR TITLE
Fix display of OSD tab save/font buttons on Android.

### PR DESCRIPTION
**Pull request by benlumley**
Before:
Screenshot 2022-06-06 at 16 16 53

After:

Screenshot 2022-06-06 at 16 33 12

The fadeIn call in osd.js causes the .content_toolbar element to end up as display: block; whereas the CSS has it display flex (and similar elements on other tabs use flex).

Fix I went for was to add a wrapper element that holds the 'supported' etc classes + gets faded in, leaving the main .content_toolbar element untampered with (ie: it remains display: flex).  Felt this is cleaner/simpler than messing with the JS to restore the flex properly after fadeIn.

Also then had to swap the order of the buttons to get them to display correctly.

### CODE REVIEW

**Review by McGiverGim — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Pull request merged**
